### ISSUE TITLE
revise the keras-yolo model for the <img> mission

### ISSUE THREAD

**Issue by MulongXie** [MulongXie/Research-ReverselyGeneratingWebCode#14]

**Comment by MulongXie** [comment]
revise the softmax out put to binary (is or not <img>)

**Comment by MulongXie** [comment]
look into the pretrained weight and erase the information of Voc mission
